css/_base.scss

```diff
@@ -51,18 +51,6 @@
             ol {
               margin-left: 20px;
             }
-
-            a {
```

Comment:
Ivy needed this for her page, can you put it back?  If you need to change link colors within your own page, you can write them under contact.scss